pkgs/development/tools/misc/d-feet/default.nix

```
@@ -89,6 +89,5 @@ python3.pkgs.buildPythonApplication rec {
     homepage = "https://wiki.gnome.org/Apps/DFeet";
     platforms = platforms.linux;
     license = licenses.gpl2;
```

```suggestion
    license = licenses.gpl2;
    maintainers = teams.gnome.members;
```

I can maintain this.